src/google/protobuf/compiler/command_line_interface.h

```diff
@@ -353,6 +358,10 @@ class LIBPROTOC_EXPORT CommandLineInterface {
   // FileDescriptorSet should be written.  Otherwise, empty.
   string descriptor_set_name_;
 
+  // If --dependency_out was given, this is the filename to which the input file
```

Comment:
I think it should be filename. Otherwise, what filename you want to use for the dependency? And what's the target name in the dependency file?